src/swapd/wallet.rs

```diff
@@ -954,6 +927,47 @@ impl Wallet {
         }
     }
 
+    pub fn create_core_arb(&mut self, swap_id: SwapId) -> Result<CoreArbitratingSetup, Error> {
+        if let Wallet::Bob(BobState {
+            bob,
+            local_params,
+            key_manager,
+            deal,
+            funding_tx,
+            remote_params,  // None
```

Comment:
This is `Some` now, right?

Reply:
good catch - indeed, otherwise, it would panic at https://github.com/lederstrumpf/farcaster-node/blob/f06edf03b4b16c3664b7e447cc47552e27c90023/src/swapd/wallet.rs#L952.

Corrected in f06edf03b4b16c3664b7e447cc47552e27c90023